cssselect/xpath.py

```diff
@@ -308,10 +308,12 @@ def xpath_attrib(self, selector):
             attrib = '@' + name
         else:
             attrib = 'attribute::*[name() = %s]' % self.xpath_literal(name)
-        if self.lower_case_attribute_values:
-            value = selector.value.lower()
+        if selector.value is None:
+            value = None
+        elif self.lower_case_attribute_values:
+            value = selector.value.value.lower()
```

Comment:
Is this fixing a bug? I don't quite get this changing to `selector.value.value`.